.changeset/gorgeous-chefs-do.md

```diff
@@ -0,0 +1,5 @@
+---
+'@journeyapps/react-native-quick-sqlite': patch
+---
+
+Fixed order of makefile flags
```

Comment:
Do we need this changeset? I don't think the order was broken before?

Reply:
Fair, should phrased fixed differently. Will remove.